numba/cpython/builtins.py

```diff
@@ -496,19 +497,25 @@ def lower_get_type_min_value(context, builder, sig, args):
             raise NotImplementedError("llvmlite only supports 32 and 64 bit floats")
         npty = getattr(np, 'float{}'.format(bw))
         res = ir.Constant(lty, -np.inf)
+    elif isinstance(typ, (types.NPDatetime, types.NPTimedelta)):
+        bw = 64
+        lty = ir.IntType(bw)
+        val = types.uint64.minval
```

Comment:
I think the minimum value for a timedelta/datetime is defined to be types.int64.minval. Is this a typo? Is this tested?